mkdocs.yml

```diff
@@ -0,0 +1,11 @@
+site_name: aws_resource_validator
```

Comment:
what does this file do?

Reply:
The mkdocs.yml file is a configuration file for MkDocs, which is used for GitHub Pages.